.github/workflows/cla.yml

```
@@ -0,0 +1,143 @@
+  LockMergedPullRequest:
+    name: "Lock merged pull request"
+    if: >-
+      github.event_name == 'pull_request_target' &&
+      github.event.action == 'closed' &&
+      github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
+    # Locking a merged PR does not need signature or repository-write access.
+    # Share the signature queue so lock and signature updates cannot race.
+    concurrency:
+      group: cla-signatures
+      queue: max
```

actionlint (1.7.12)

[error] unexpected key "queue" for "concurrency" section. expected one of "cancel-in-progress", "group"

(syntax-check)